src/flatc.cpp

```diff
@@ -360,6 +371,17 @@ int main(int argc, const char *argv[]) {
               printf("%s\n", flatbuffers::WordWrap(
                   make_rule, 80, " ", " \\").c_str());
           }
+          if(grpc_enabled){
```

Comment:
whitespace formatting of if/else, please check Google C++ Style Guide, or use clang-format (on only your changes) withe the Google style.